### PR TITLE
CONTRIBUTING.md : adds detail around internal linking for translations

### LINKED ISSUE #354: Update CONTRIBUTING notes to explain how to make non-duplicate anchor tags for non-ASCII languages
https://github.com/bitcoinops/bitcoinops.github.io/pull/349#issuecomment-590011688

### PR DESCRIPTION
**Pull request by bitschmidty**
Closes #354

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -24,6 +24,12 @@ standardized:
   - For blog posts, place the file in `_posts/<language code>/`
 - Set the `lang` field to `<language code>`
 - Append `-<language code>` to both the `slug` and `name` fields
+- When linking internally, link to the appropriate translation, if one is available.
```
Comment (jnewbery) [inline CONTRIBUTING.md]: s/linking internally/linking to material on the Optech site/ for clarity.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -24,6 +24,12 @@ standardized:
...
+  - When linking to a specific header/bullet within a page, #anchor links are [automatically generated](https://github.com/bitcoinops/bitcoinops.github.io/blob/master/_plugins/auto-anchor.rb).
+      - Due to [limitations](https://github.com/bitcoinops/bitcoinops.github.io/pull/349)
+        in the automatic anchoring process, some languages [need HTML comments
+        inserted](https://github.com/bitcoinops/bitcoinops.github.io/commit/4e450d1a1f72219ec50ad91edae605647164d25d#diff-435f99f277721eff9e2f244149575f41R41)
```
Comment (jnewbery) [inline CONTRIBUTING.md]: I suggest "some languages need HTML comments inserted. See for example the `<!--1-->` [comment](https://github.com/bitcoinops/bitcoinops.github.io/commit/4e450d1a1f72219ec50ad91edae605647164d25d#diff-435f99f277721eff9e2f244149575f41R41) in this Japanese newsletter." to be absolutely clear about what you're demonstrating in the link.

Comment (bitschmidty) [reply CONTRIBUTING.md]: I think that’s clearer, thanks for the suggestion. Changed.

File: CONTRIBUTING.md (hunk 1 above)
Comment (bitschmidty) [reply CONTRIBUTING.md]: Changed.

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -24,6 +24,14 @@ standardized:
   - For blog posts, place the file in `_posts/<language code>/`
 - Set the `lang` field to `<language code>`
 - Append `-<language code>` to both the `slug` and `name` fields
+- When linking to material on the Optech site, link to the appropriate
+  translation, if one is available.
+  - When linking to a specific header/bullet within a page, #anchor links are [automatically generated](https://github.com/bitcoinops/bitcoinops.github.io/blob/master/_plugins/auto-anchor.rb).
+      - Due to [limitations](https://github.com/bitcoinops/bitcoinops.github.io/pull/349)
+        in the automatic anchoring process, some languages need HTML comments
+        inserted into the header/bullet to ensure a unique anchor is generated.
+        For example, see the <!--1--> [comment](https://github.com/bitcoinops/bitcoinops.github.io/commit/4e450d1a1f72219ec50ad91edae605647164d25d#diff-435f99f277721eff9e2f244149575f41R41)
```
Comment (jnewbery) [inline CONTRIBUTING.md]: oops. Looks like this comment is commented out in the markdown. I think you can surround it with backticks to get it to display.

Comment (bitschmidty) [reply CONTRIBUTING.md]: Missed that you’re right. Fixup commit pushed.

**Comment by jnewbery** [comment]
Thanks Schmidty

**Pull request merged**